### PR TITLE
Add missing filter translations

### PR DESCRIPTION
**Pull request by datune**
<a href="https://gitpod.io/#https://github.com/wintercms/winter/pull/303"></a>

### CODE REVIEW

File: modules/system/lang/de/client.php [hunk 1]
```diff
@@ -61,7 +61,11 @@
 
     'filter' => [
         'group' => [
-            'all' => 'Alle'
+            'all' => 'all',
```
Comment (bennothommo) [inline modules/system/lang/de/client.php]: @datune Is this meant to be "alle"? I'm probably wrong, but just wanted to check :)

Would you also mind changing all "Alle" references in other `all` definitions, as these are all supposed to be lower-case?

Comment (datune) [reply modules/system/lang/de/client.php]: You‘re indeed wrong 😉 It is supposed to be like that.

Comment (bennothommo) [reply modules/system/lang/de/client.php]: I sometimes am 😂 

Do you mind doing this for us?:
> Would you also mind changing all "Alle" references in other all definitions, as these are all supposed to be lower-case?

Comment (datune) [reply modules/system/lang/de/client.php]: Thanks for catching that one, I've updated the PR!

**Comment by bennothommo** [comment]
Thanks @datune :)

**Pull request merged**